### PR TITLE
Plugins fitbot

### PR DESCRIPTION
**Pull request by dinowais**
@shahidash

### CODE REVIEW

File: will/plugins/fitbot/custom_messages.py [hunk 1]
```diff
@@ -46,46 +47,55 @@ def hear_awo(self, message):
...
+    #     self.say("@all Group Stats 12:56", notify=True, color='green')
+    #     self.say(rendered_template("group_stats.html", context), notify=True,
+    #              color='green', html=True)
+    #
+    # @periodic(hour='9', minute='45', day_of_week="mon-fri")
+    # def schedule_945_message(self):
+    #     context = group_stats()
+    #     self.say("@all Group Stats 9:45", notify=True, color='green')
+    #     self.say(rendered_template("group_stats.html", context), notify=True,
+    #              color='green', html=True)
```
Comment (shahidash) [inline will/plugins/fitbot/custom_messages.py]: remove commented  code

File: will/settings.py [hunk 2]
```diff
@@ -1,10 +1,15 @@
 import os
-from utils import show_valid, warn, note
-from clint.textui import puts, indent
 from urlparse import urlparse
 
+from clint.textui import puts, indent
+
+from utils import show_valid, warn, note
+
```
Comment (shahidash) [inline will/settings.py:7]: follow pep8

**Review by shahidash — APPROVED** [review]
ok merge ahead

File: config.py [hunk 3]
```diff
@@ -95,7 +95,7 @@
 
 # The room will will talk to if the trigger is a webhook and he isn't told a specific room.
 # Default is the first of ROOMS.
-DEFAULT_ROOM = 'AW'
+DEFAULT_ROOM = 'DailyStandup'
```
Comment (shahidash) [inline config.py:98]: Handle this for local setting

Comment (dinowais) [reply config.py:98]: @shahidash its in config so we can overide in local settings

**Review by shahidash — APPROVED** [review]

**Review by shahidash — APPROVED** [review]

**Review by shahidash — APPROVED** [review]

**Review by shahidash — APPROVED** [review]

**Comment by shahidash** [comment]
👍

**Pull request merged**